x/profile/types/key_validator_by_address.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+import "encoding/binary"
+
+var _ binary.ByteOrder
+
+const (
+	// ValidatorKeyPrefix is the prefix to retrieve all Validator
+	ValidatorKeyPrefix = "Validator/value/"
+)
+
+// ValidatorKey returns the store key to retrieve a Validator from the index fields
+func ValidatorKey(
+	address string,
```

Comment:
Just a general observation, I think we can use multiline arguments when there are more than 3 or 4 args otherwise we can use regular single liner for better code readability.

Reply:
Sure. Totally agree!!

Reply:
This is scaffolded by starport. Maybe it's better we also handle this behavior into the starport too?

Reply:
Agree.
I put it this way in the template because I find it cleaner to have a plush for statement on two lines rather than one

I think we can a general review of those details at the end of the milestone, removing blank lines, formating proto files, etc... (that of course doesn't prevent us to do it on the fly when we spot those details in the other PRs)